threedsecure/client.go

```diff
@@ -13,24 +13,28 @@ type Client struct {
 	Key string
 }
 
-// New POSTs new 3D Secure auths.
-// For more details see https://stripe.com/docs/api#create_three_d_secure.
+// This package is deprecated and should not be used anymore.
+// It is still here for backwards compatibility for now.
+// To use 3D Secure, please use the source package instead.
```

Comment:
Minor, but in Go you can actually document the packages directly. Would you mind moving this documentation up on top of `package theedsecure` above? That way it'll show up in rendered Godoc, etc.